app/src/view/screens/Sell.js

```
@@ -1191,88 +1209,95 @@ export default function Sell({navigation}) {
+              }}
+            >
               Join Us Today
             </Text>
 
             <Text
               style={{
-                color: '#9597A6',
+                color: "#9597A6",
                 marginTop: hp(0.5),
-                textAlign: 'center',
+                textAlign: "center",
```

Biome (1.9.4)

[error] This property value named marginTop is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named marginTop


(lint/suspicious/noDuplicateObjectKeys)